--- a/Makefile.in	2009-06-30 14:50:13.287235839 -0700
+++ b/Makefile.in	2009-06-30 14:53:42.785181192 -0700
@@ -38,9 +38,8 @@
 @INCLUDE_HSIEH_SRC_TRUE@am__append_2 = hsieh_hash.c
 @BUILD_BYTEORDER_TRUE@am__append_3 = byteorder.c
 subdir = libmemcached
-DIST_COMMON = $(am__pkginclude_HEADERS_DIST) $(noinst_HEADERS) \
-	$(srcdir)/Makefile.am $(srcdir)/Makefile.in \
-	$(srcdir)/libmemcached_config.h.in
+DIST_COMMON = $(am__noinst_HEADERS_DIST) $(srcdir)/Makefile.am \
+	$(srcdir)/Makefile.in $(srcdir)/libmemcached_config.h.in
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
 am__aclocal_m4_deps = $(top_srcdir)/m4/extensions.m4 \
	$(top_srcdir)/m4/pod2man.m4 $(top_srcdir)/m4/debug.m4 \
@@ -54,15 +53,7 @@
 mkinstalldirs = $(install_sh) -d
 CONFIG_HEADER = libmemcached_config.h
 CONFIG_CLEAN_FILES =
-am__vpath_adj_setup = srcdirstrip=`echo "$(srcdir)" | sed 's|.|.|g'`;
-am__vpath_adj = case $$p in \
-    $(srcdir)/*) f=`echo "$$p" | sed "s|^$$srcdirstrip/||"`;; \
-    *) f=$$p;; \
-  esac;
-am__strip_dir = `echo $$p | sed -e 's|^.*/||'`;
-am__installdirs = "$(DESTDIR)$(libdir)" "$(DESTDIR)$(pkgincludedir)"
-libLTLIBRARIES_INSTALL = $(INSTALL)
-LTLIBRARIES = $(lib_LTLIBRARIES)
+LTLIBRARIES = $(noinst_LTLIBRARIES)
 am__DEPENDENCIES_1 =
 am__libmemcached_la_SOURCES_DIST = crc.c memcached.c memcached_auto.c \
	memcached_analyze.c memcached_behavior.c memcached_callback.c \
@@ -107,13 +98,10 @@
	$(LDFLAGS) -o $@
 SOURCES = $(libmemcached_la_SOURCES)
 DIST_SOURCES = $(am__libmemcached_la_SOURCES_DIST)
-am__pkginclude_HEADERS_DIST = memcached.h memcached.hh \
-	memcached_constants.h memcached_get.h memcached_result.h \
-	memcached_server.h memcached_storage.h memcached_string.h \
-	memcached_types.h memcached_watchpoint.h memcached_util.h \
-	memcached_pool.h
-pkgincludeHEADERS_INSTALL = $(INSTALL_HEADER)
-HEADERS = $(noinst_HEADERS) $(pkginclude_HEADERS)
+am__noinst_HEADERS_DIST = libmemcached_probes.h libmemcached_config.h \
+	memcached_io.h common.h memcached/protocol_binary.h \
+	@DTRACE_HEADER@ memcached_util.h memcached_pool.h
+HEADERS = $(noinst_HEADERS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
@@ -247,17 +235,10 @@
 EXTRA_DIST = libmemcached_probes.d memcached/README.txt libmemcached.ver
 BUILT_SOURCES = @DTRACE_HEADER@
 EXTRA_HEADERS =
-noinst_HEADERS = libmemcached_probes.h \
-                 libmemcached_config.h \
-		 memcached_io.h \
-                 common.h \
-                 memcached/protocol_binary.h @DTRACE_HEADER@
-
-pkginclude_HEADERS = memcached.h memcached.hh memcached_constants.h \
-	memcached_get.h memcached_result.h memcached_server.h \
-	memcached_storage.h memcached_string.h memcached_types.h \
-	memcached_watchpoint.h $(am__append_1)
-lib_LTLIBRARIES = libmemcached.la
+noinst_HEADERS = libmemcached_probes.h libmemcached_config.h \
+	memcached_io.h common.h memcached/protocol_binary.h \
+	@DTRACE_HEADER@ $(am__append_1)
+noinst_LTLIBRARIES = libmemcached.la
 libmemcached_la_SOURCES = crc.c memcached.c memcached_auto.c \
	memcached_analyze.c memcached_behavior.c memcached_callback.c \
	memcached_connect.c memcached_delete.c memcached_do.c \
@@ -323,35 +304,17 @@

 distclean-hdr:
	-rm -f libmemcached_config.h stamp-h1
-install-libLTLIBRARIES: $(lib_LTLIBRARIES)
-	@$(NORMAL_INSTALL)
-	test -z "$(libdir)" || $(MKDIR_P) "$(DESTDIR)$(libdir)"
-	@list='$(lib_LTLIBRARIES)'; for p in $$list; do \
-	  if test -f $$p; then \
-	    f=$(am__strip_dir) \
-	    echo " $(LIBTOOL) --mode=install $(libLTLIBRARIES_INSTALL) $(INSTALL_STRIP_FLAG) '$$p' '$(DESTDIR)$(libdir)/$$f'"; \
-	    $(LIBTOOL) --mode=install $(libLTLIBRARIES_INSTALL) $(INSTALL_STRIP_FLAG) "$$p" "$(DESTDIR)$(libdir)/$$f"; \
-	  else :; fi; \
-	done
-
-uninstall-libLTLIBRARIES:
-	@$(NORMAL_UNINSTALL)
-	@list='$(lib_LTLIBRARIES)'; for p in $$list; do \
-	  p=$(am__strip_dir) \
-	  echo " $(LIBTOOL) --mode=uninstall rm -f '$(DESTDIR)$(libdir)/$$p'"; \
-	  $(LIBTOOL) --mode=uninstall rm -f "$(DESTDIR)$(libdir)/$$p"; \
-	done

-clean-libLTLIBRARIES:
-	-test -z "$(lib_LTLIBRARIES)" || rm -f $(lib_LTLIBRARIES)
-	@list='$(lib_LTLIBRARIES)'; for p in $$list; do \
+clean-noinstLTLIBRARIES:
+	-test -z "$(noinst_LTLIBRARIES)" || rm -f $(noinst_LTLIBRARIES)
+	@list='$(noinst_LTLIBRARIES)'; for p in $$list; do \
	  dir="`echo $$p | sed -e 's|/[^/]*$$||'`"; \
	  test "$$dir" != "$$p" || dir=.; \
	  echo "rm -f \"$${dir}/so_locations\""; \
	  rm -f "$${dir}/so_locations"; \
	done
 libmemcached.la: $(libmemcached_la_OBJECTS) $(libmemcached_la_DEPENDENCIES)
-	$(libmemcached_la_LINK) -rpath $(libdir) $(libmemcached_la_OBJECTS) $(libmemcached_la_LIBADD) $(LIBS)
+	$(libmemcached_la_LINK)  $(libmemcached_la_OBJECTS) $(libmemcached_la_LIBADD) $(LIBS)

 mostlyclean-compile:
	-rm -f *.$(OBJEXT)
@@ -421,23 +384,6 @@

 clean-libtool:
	-rm -rf .libs _libs
-install-pkgincludeHEADERS: $(pkginclude_HEADERS)
-	@$(NORMAL_INSTALL)
-	test -z "$(pkgincludedir)" || $(MKDIR_P) "$(DESTDIR)$(pkgincludedir)"
-	@list='$(pkginclude_HEADERS)'; for p in $$list; do \
-	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
-	  f=$(am__strip_dir) \
-	  echo " $(pkgincludeHEADERS_INSTALL) '$$d$$p' '$(DESTDIR)$(pkgincludedir)/$$f'"; \
-	  $(pkgincludeHEADERS_INSTALL) "$$d$$p" "$(DESTDIR)$(pkgincludedir)/$$f"; \
-	done
-
-uninstall-pkgincludeHEADERS:
-	@$(NORMAL_UNINSTALL)
-	@list='$(pkginclude_HEADERS)'; for p in $$list; do \
-	  f=$(am__strip_dir) \
-	  echo " rm -f '$(DESTDIR)$(pkgincludedir)/$$f'"; \
-	  rm -f "$(DESTDIR)$(pkgincludedir)/$$f"; \
-	done

 ID: $(HEADERS) $(SOURCES) $(LISP) $(TAGS_FILES)
	list='$(SOURCES) $(HEADERS) $(LISP) $(TAGS_FILES)'; \
@@ -518,9 +464,6 @@
	$(MAKE) $(AM_MAKEFLAGS) check-am
 all-am: Makefile $(LTLIBRARIES) $(HEADERS) libmemcached_config.h
 installdirs:
-	for dir in "$(DESTDIR)$(libdir)" "$(DESTDIR)$(pkgincludedir)"; do \
-	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
-	done
 install: $(BUILT_SOURCES)
	$(MAKE) $(AM_MAKEFLAGS) install-am
 install-exec: install-exec-am
@@ -549,7 +492,7 @@
	-test -z "$(BUILT_SOURCES)" || rm -f $(BUILT_SOURCES)
 clean: clean-am

-clean-am: clean-generic clean-libLTLIBRARIES clean-libtool \
+clean-am: clean-generic clean-libtool clean-noinstLTLIBRARIES \
	mostlyclean-am

 distclean: distclean-am
@@ -568,11 +511,11 @@

 info-am:

-install-data-am: install-pkgincludeHEADERS
+install-data-am:

 install-dvi: install-dvi-am

-install-exec-am: install-libLTLIBRARIES
+install-exec-am:

 install-html: install-html-am

@@ -604,26 +547,34 @@

 ps-am:

-uninstall-am: uninstall-libLTLIBRARIES uninstall-pkgincludeHEADERS
+uninstall-am:

 .MAKE: install-am install-strip

 .PHONY: CTAGS GTAGS all all-am check check-am clean clean-generic \
-	clean-libLTLIBRARIES clean-libtool ctags distclean \
+	clean-libtool clean-noinstLTLIBRARIES ctags distclean \
	distclean-compile distclean-generic distclean-hdr \
	distclean-libtool distclean-tags distdir dvi dvi-am html \
	html-am info info-am install install-am install-data \
	install-data-am install-dvi install-dvi-am install-exec \
	install-exec-am install-html install-html-am install-info \
-	install-info-am install-libLTLIBRARIES install-man install-pdf \
-	install-pdf-am install-pkgincludeHEADERS install-ps \
-	install-ps-am install-strip installcheck installcheck-am \
-	installdirs maintainer-clean maintainer-clean-generic \
-	mostlyclean mostlyclean-compile mostlyclean-generic \
-	mostlyclean-libtool pdf pdf-am ps ps-am tags uninstall \
-	uninstall-am uninstall-libLTLIBRARIES \
-	uninstall-pkgincludeHEADERS
-
+	install-info-am install-man install-pdf install-pdf-am \
+	install-ps install-ps-am install-strip installcheck \
+	installcheck-am installdirs maintainer-clean \
+	maintainer-clean-generic mostlyclean mostlyclean-compile \
+	mostlyclean-generic mostlyclean-libtool pdf pdf-am ps ps-am \
+	tags uninstall uninstall-am
+
+	         memcached.h \
+	    	 memcached.hh \
+		 memcached_constants.h \
+		 memcached_get.h \
+		 memcached_result.h \
+		 memcached_server.h \
+		 memcached_storage.h \
+		 memcached_string.h \
+		 memcached_types.h \
+		 memcached_watchpoint.h

 dtrace_probes.h: libmemcached_probes.d
	$(DTRACE) $(DTRACEFLAGS) -o dtrace_probes.tmp -h -s libmemcached_probes.d
